clientlib/src/main/proto/yelp/nrtsearch/luceneserver.proto

```diff
@@ -435,6 +435,7 @@ enum FieldType {
     _ID = 12; // Field which will be used as document IDs
     POLYGON = 13; // Geojson string for polygon
     OBJECT = 14; // JSON
+    VECTOR = 15; // List of float values
```

Comment:
nit: place the comment on the line above the field, otherwise it doesn't end up in the auto generated documentation. Yeah, it isn't done right in most places.